indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -41,9 +42,19 @@
     <div class="section">
         <div class="icon icon-wrench"></div>
         <div class="toolbar right">
-            <a class="i-button js-reset-btn icon-close warning" type="button">
-                {%- trans %}Clear filters{% endtrans -%}
-            </a>
+            <div class="group left">
+                <a class="i-button icon-checkbox-checked arrow left icon-only" aria-hidden="true" href="#"
```

Comment:
Maybe you can simplify the logic within `setupRegistrationListFilter` by using the recently added `js-dropdown`. Could you check?